tests/tcl/tests/test_helper.tcl

```diff
@@ -60,6 +60,7 @@ set ::all_tests {
     unit/scripting
     integration/slotmigrate
     integration/slotimport
+    integration/redis-cli
```

Comment:
Maybe we need change configuration of daily CI since `integration/redis-cli` is mandatory now?

Reply:
I don't check logics in daily CI. How can it be related?

Reply:
OK. I can see the related code. Let me push a new commit.